docs/pages/define/connectiontypes/db.bigquery.md

```diff
@@ -0,0 +1,111 @@
+{% include navigation.html %}
+
+# db.bigquery
+
+This type connects to Google BigQuery. It uses the Magnitude Simba drivers for BigQuery. More information can be found [here](https://cloud.google.com/bigquery/providers/simba-drivers). More technical information can be found [here](https://www.simba.com/drivers/bigquery-odbc-jdbc/).
+
+*Important* Only the Simba driver version 4.2 is supported for now.
+
+Get more information at [https://cloud.google.com/bigquery](https://cloud.google.com/bigquery).
+
+## Use cases
+
+* Connect to database
+
+## Parameters
+
+### 1: host
+
+`host: "host name"`
+* define the host name of the server
+
+### 2: port
+
+`port: "port number"`
+* define the port number for connecting to the host
+
+### 3: project
+
+`project: "project name"`
+* define the name of the Google Cloud project
+
+### 4: dataset
+
+`dataset: "dataset name"`
+* define the name of the dataset
+
+### 4: authValue
+
+`authValue: "authValue number"`
+* define the number that specifies the type of authentication used by the driver
+  * 0: The driver uses service-based OAuth authentication
+  * 1: The driver uses user-based OAuth authentication
+  * 2: The driver uses pre-generated tokens for authentication
+  * 3: The driver uses Application Default Credentials for authentication
```

Comment:
maybe rename to authMode and use more functional options like 'service', 'user', 'token' and 'application default credentials' instead of 0, 1, 2, 3. This keeps it more in line with Oracle as well, cfr. 'mode' and 'tnsAlias'/'service'

Reply:
changed to authMode with options: service, user, token, default